docs/designers-developers/designers/block-design.md

```diff
@@ -68,7 +74,13 @@ Avoid long, multi-line block names.
 
 ### Block Description
 
-Every block should include a description in the “Block” tab of the Settings sidebar. This description should explain your block's function clearly. Keep it to a single sentence.
+Every block should include a description in the “Block” tab of the Settings sidebar. This description should explain your block's function clearly.
```

Comment:
The question that arises and not explained here (nor previously) is "How do I specify my description in the Settings sidebar?" 

What do you think of this addition? I added a link to the `registerBlockType` function which does unfortunately say the description field is optional.

```suggestion
Every block should include a description attribute in the [registerBlockType function](/docs/designers-developers/developers/block-api/block-registration/). The description will display in the Settings sidebar and should explain your block's function clearly.
```

Reply:
Oh, great point! Absolutely agree. 🥇 

As a side note, definitely feels description should be non-optional, but at least better docs should help encourage people to make use of this field.